demo/simple_demo.html

```diff
@@ -9,14 +9,55 @@
     <link rel="import" href="../../polymer/polymer.html" />
     <link rel="import" href="../../px-theme/px-theme-styles.html">
     <style include="px-theme-styles" is="custom-style"></style>
+<link rel="import" href="../px-datetime-picker.html" />
 
-    <link rel="import" href="../px-datetime-picker.html" />
+</head>
 
-  </head>
+<body>
+  <dom-bind>
+    <template is="dom-bind">
+      <h2>px-datetime-picker</h2>
+      <div id="fit" style="margin-left: 50px; width: 500px; height: 600px; background-color:aliceblue">
+      <px-datetime-picker
+      date-format="YYYY/MM/DD"
+      time-format="HH:mm:ss"
+      date-time="2016-02-05T05:30:00.000Z"
+      show-buttons
+      fill-container
+      fit-into-element="[[getElement('#fit')]]"
+      collapse-at="60">
```

Comment:
I had to use a larger value (like 360) in order to actually see this trigger in my local.